### PR TITLE
bugfix to include missing hanadbname field

### PR DESCRIPTION
**Pull request by pabowers**
Rebased on top of Laura's branch to avoid conflicts

### CODE REVIEW

**Review by lagalbra — CHANGES_REQUESTED** [review]
azext_hanaonazure/azext_metadata.json also needs to be updated with the new version

File: azext_hanaonazure/azext_metadata.json [hunk 1]
```diff
@@ -1,4 +1,4 @@
 {
   "azext.minCliCoreVersion": "2.0.46",
-  "version": "0.3.9"
+  "version": "0.4.0"
```
Comment (PakDLiu) [inline azext_hanaonazure/azext_metadata.json]: nit: i would do 3.10. going to 4.0 implies big changes. maybe when we ship, we can change it to 4.0

**Review by PakDLiu — APPROVED** [review]

File: azext_hanaonazure/azext_metadata.json (hunk 1 above)
Comment (lagalbra) [reply azext_hanaonazure/azext_metadata.json]: +1 I like this idea; test to see if this works ok. https://microsoft.sharepoint.com/teams/AzureDedicatedEngineeringTeam/_layouts/OneNote.aspx?id=%2Fteams%2FAzureDedicatedEngineeringTeam%2FSiteAssets%2FAzure%20Dedicated%20Engineering%20Team%20Notebook&wd=target%28SAP%20HANA.one%7CFB7D0654-1276-104F-AF53-BA55FC4DA39E%2FAzure%20CLI%7C7303BF06-991D-4331-9BF8-542F65E5D86E%2F%29

Comment (pabowers) [reply azext_hanaonazure/azext_metadata.json]: done and tested locally

**Review by lagalbra — APPROVED** [review]

File: azext_hanaonazure/azext_metadata.json [hunk 2]
```diff
@@ -1,4 +1,4 @@
 {
   "azext.minCliCoreVersion": "2.0.46",
-  "version": "0.3.9"
```
Comment (persiaAziz-zz) [inline azext_hanaonazure/azext_metadata.json]: Is this a generated file as well? Why does this repo have versions in two places?

Comment (pabowers) [reply azext_hanaonazure/azext_metadata.json]: Not generated, and out of the scope of this PR for why there are versions in several places. I agree it is confusing, but it isn't something I'm going to address at the moment.

**Pull request merged**